code/modules/security_levels/security_levels.dm

```diff
@@ -60,6 +55,8 @@
 						FA.update_icon()
 				for(var/obj/machinery/computer/shuttle/pod/pod in machines)
 					pod.admin_controlled = 0
+		for(var/mob/living/silicon/robot/R in living_mob_list)

```

Comment:
atleast use the silicon mobs list, still typed tho cos that has AIs in it and stuff (just a smaller list so less looping)